openapi.json

```diff
@@ -1906,6 +1847,74 @@
         ]
       }
     },
+    "/v0/pipelines/{pipeline_name}/tables/{table_name}/connectors/{connector_name}/{action}": {
+      "post": {
+        "tags": [
+          "Pipelines"
+        ],
+        "summary": "Start (resume) or pause the input connector.",
+        "description": "The following values of the `action` argument are accepted: `start` and `pause`.\n\nInput connectors can be in either the `Running` or `Paused` state. By default,\nconnectors are initialized in the `Running` state when a pipeline is deployed.\nIn this state, the connector actively fetches data from its configured data\nsource and forwards it to the pipeline. If needed, a connector can be created\nin the `Paused` state by setting its\n[`paused`](https://docs.feldera.com/connectors/#generic-attributes) property\nto `true`. When paused, the connector remains idle until reactivated using the\n`start` command. Conversely, a connector in the `Running` state can be paused\nat any time by issuing the `pause` command.\n\nThe current connector state can be retrieved via the\n`GET /v0/pipelines/{pipeline_name}/stats` endpoint.\n\nNote that only if both the pipeline *and* the connector state is `Running`,\nis the input connector active.\n```text\nPipeline state    Connector state    Connector is active?\n--------------    ---------------    --------------------\nPaused            Paused             No\nPaused            Running            No\nRunning           Paused             No\nRunning           Running            Yes\n```",
```

Comment:
can we check if this renders ok on docs.feldera.com/api?

Reply:
It renders well:
![docs](https://github.com/user-attachments/assets/88d10d7b-f593-469b-90bd-be360cf3e9a9)